components/developer/clang-90/patches/08-gas.patch

```diff
@@ -0,0 +1,11 @@
+--- llvm-8.0.1.src/tools/clang/lib/Driver/ToolChains/Solaris.cpp.1	2019-07-31 09:34:25.331931850 +0000
++++ llvm-8.0.1.src/tools/clang/lib/Driver/ToolChains/Solaris.cpp	2019-07-31 09:38:22.787167485 +0000
+@@ -40,7 +40,7 @@
+   for (const auto &II : Inputs)
+     CmdArgs.push_back(II.getFilename());
+ 
+-  const char *Exec = Args.MakeArgString(getToolChain().GetProgramPath("as"));
++  const char *Exec = Args.MakeArgString(getToolChain().GetProgramPath("gas"));
```

Comment:
just curious, SPARC uses GNU as now? All IX86 targets supported by clang have long since used the integrated assembler, this statement has no effect as a result

Reply:
Perhaps, this can be dropped, but this needs some testing.